.circleci/config.yml

```diff
@@ -31,6 +31,13 @@ jobs:
             wget https://github.com/mikefarah/yq/releases/download/3.3.2/yq_linux_amd64
             chmod a+x yq_linux_amd64
             sudo mv yq_linux_amd64 /usr/bin/yq
+      - run:
+          name: Install git-secrets
```

Comment:
It looks like you added an "Install git-secrets" section to this OpenAPI validation step, but no other steps use it. Was this supposed to be removed? Or is it used by the Maven stuff?

Reply:
Yeah, this PR makes having git-secrets on the path a build requirement even if it doesn't do the scan.